assets/terraform-modules/aws/flatcar-linux/kubernetes/variables.tf

```diff
@@ -185,3 +184,9 @@ variable "kube_apiserver_extra_flags" {
   type        = list(string)
   default     = []
 }
+
+variable "encrypt_pod_traffic" {
+  description = "Enable in-cluster pod traffic."
```

Comment:
This description doesn't mention encryption. If you fix this, please fix all occurrences.